src/extension/background-script/connectors/lnd.ts

```diff
@@ -85,24 +84,16 @@ class Lnd implements Connector {
     });
   }
   async keySend(args: KeysendArgs): Promise<SendPaymentResponse> {
-    //See: https://gist.github.com/dellagustin/c3793308b75b6b0faf134e64db7dc915
```

Comment:
Why not leave this comment in? I added it to make clear where I got this code sample from.

Reply:
ups. of course.